seedfarmer/mgmt/build_info.py

```diff
@@ -40,5 +41,10 @@ def get_build_env_params(build_ids: List[str], session: Optional[Session] = None
     return dict(ChainMap(*list_of_envs)) if list_of_envs else None
 
 
-def get_manifest_schema() -> Dict[str, Any]:
-    return DeploymentManifest.model_json_schema()
+def get_manifest_schema(type: str) -> Optional[Dict[str, Any]]:
+    if "deployment" == type.lower():
+        return DeploymentManifest.model_json_schema()
+    elif "module" == type.lower():
+        return ModuleManifest.model_json_schema()
+    else:
+        return None
```

Comment:
Minor nitpick: We could just raise an error here, and that way wouldn't need to change the return type of the function.

Reply:
hmmm...will check